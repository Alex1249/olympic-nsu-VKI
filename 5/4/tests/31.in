25
t o z a l s n c q g b y i r u k h p x v w m j f e
