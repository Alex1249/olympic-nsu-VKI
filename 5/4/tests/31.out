t o z a l s n c q g b y i r u k h p x w e f j m v
